conf/master

```diff
@@ -234,6 +234,9 @@
 # Set the ZeroMQ high water marks
 # http://api.zeromq.org/3-2:zmq-setsockopt
 
+# The listen queue size / backlog
+#zmq_backlog: 4096
```

Comment:
With this commented out, let's set this to the same value that we have in the opts dictionary, so there's no confusion about the default value is.